pkg/action/diff.go

```diff
@@ -190,7 +217,13 @@ func Diff(ctx context.Context, c malcontent.Config) (*malcontent.Report, error)
 		if srcFile != nil && destFile != nil {
 			formatSrc := displayPath(srcBase, srcFile.Path)
 			formatDest := displayPath(destBase, destFile.Path)
-			handleFile(ctx, c, srcFile, destFile, fmt.Sprintf("%s -> %s", formatSrc, formatDest), d)
+			if scoreFile(srcFile, destFile) {
```

Comment:
This was the last remaining bug. Previously, we were hitting `handleFile` for the files we calculate moves for, so they would just show up as `changed` rather than `moved`. Now, we'll handle those files correctly via `inferMoves` while sending every other file to `handleFile`.